Plots/Meteograms/NCL_meteo_1.py

```diff
@@ -0,0 +1,291 @@
+
+# Create two inset axes in the second axes
+axin1 = ax2.inset_axes([0.0, 0.3, 1, 0.30])
+axin2 = ax2.inset_axes([0.0, -.1, 1, 0.35])
+
+# Plot bar chart
+
+# Set aspect ratio of inset axis
+axin1.set_aspect(30)
+
+# Plot bars depicting the rain03 variable
+axin1.bar(taus,
+          rain03,
+          width=3,
+          color='limegreen',
+          edgecolor='k',
+          linewidth=.2)
+
+# Use the geocat.viz function to set the y axis label
+gvutil.set_titles_and_labels(axin1,
+                             ylabel='3hr rain total',
+                             labelfontsize=12)
+
+# Determine the labels for each tick on the x and y axes
+yticklabels = ['0.0', '0.10', '0.20', '0.30', '0.40', '0.50']
+xticklabels = ['12z', '', '18z', '', 'Apr29', '',
+               '06z', '', '12z', '', '18z', '',
+               'Apr30', '', '06z', '', '12z', '',
+               '18z', '', 'May01', '', '06z', '', '12z']
+
+# Use the geocat.viz function to set axes limits and ticks
+gvutil.set_axes_limits_and_ticks(axin1,
+                                 xlim=[0, 72],
+                                 ylim=[0, .5],
+                                 xticks=np.arange(0, 75, 3),
+                                 yticks=np.arange(0, .6, 0.1),
+                                 xticklabels=xticklabels,
+                                 yticklabels=yticklabels)
+
+# Use the geocat.viz function to add minor ticks
+gvutil.add_major_minor_ticks(axin1,
+                             y_minor_per_major=5,
+                             labelsize="small")
+
+# Make ticks only show up on bottom, right, and left of inset axis
+axin1.tick_params(bottom=True, left=True, right=True, top=False)
+axin1.tick_params(which='minor', top=False)
+
+# Plot line chart
+
+# Set aspect ratio of inset axis
+axin2.set_aspect(2)
+
+# Plot lines depicting the tempht variable
+axin2.plot(taus, tempht, color='red')
+
+# Use the geocat.viz function to set the y axis label
+gvutil.set_titles_and_labels(axin2,
+                             ylabel='Temp at 2m',
+                             labelfontsize=12)
+
+# Determine the labels for each tick on the x and y axes
+yticklabels = ['59.0', '60.0', '61.0', '62.0', '63.0', '64.0']
+xticklabels = ['12z', '', '18z', '', 'Apr29', '',
+               '06z', '', '12z', '', '18z', '',
+               'Apr30', '', '06z', '', '12z', '',
+               '18z', '', 'May01', '', '06z', '', '12z']
+
+# Use the geocat.viz function to set inset axes limits and ticks
+gvutil.set_axes_limits_and_ticks(axin2,
+                                 xlim=[0, 72],
+                                 ylim=[59, 64.5],
+                                 xticks=np.arange(0, 75, 3),
+                                 yticks=np.arange(59, 65),
+                                 xticklabels=xticklabels,
+                                 yticklabels=yticklabels)
+
+# Use the geocat.viz function to add minor ticks
+gvutil.add_major_minor_ticks(axin2,
```

Comment:
The temperature plot doesn't have minor xticks. You can remove them by specifying `x_minor_per_major=1`